### PR TITLE
feat: helm Parameters/Variables/Values.yaml can be configured via XML/DSL

### PR DESCRIPTION
**Pull request by manusa**
## Description
feat: helm Parameters/Variables/Values.yaml can be configured via XML/DSL

Instead of providing parameters via `Template` resource fragment, they can now be provided via pom.xml XML configuration or build.gadle DSL configuration.

The following configuration:
```xml
<helm>
  <parameters>
    <parameter>
      <name>chart_name</name>
      <value>{{ .Chart.Name }}</value>
    </parameter>
  </parameters>
</helm>
```
Can interpolate a resource fragment:
```yaml
metadata:
 annotations:
   chart/name: ${chart_name}
```
To produce a:
```yaml
apiVersion: v1
kind: #...
metadata:
  annotations:
   chart/name: {{ .Chart.Name }}
# ...
```
## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [x] I have implemented unit tests to cover my changes
 - [x] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

/cc @dloiacono

### CODE REVIEW

**Comment by dloiacono** [comment]
great functionality!
Thanks @manusa

**Review by rohanKanojia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Usage:
 * Fix #1142: Log informative message whenever a goal/task is skipped
 * Fix #1197: Kubernetes Gradle Build tasks don't account for image model configuration skip field
 * Fix #1245: Deprecate `maven.jkube.io` annotation prefix used in enrichers in favor of `jkube.eclipse.org`
+* Fix #1244: Helm Parameters/Variables/Values.yaml can be configured via XML/DSL
```
Comment (sunix) [inline CHANGELOG.md:53]: is it worth adding "feat: Helm supports placeholders/variables with dotted notation" in the changelog ?

**Review by sunix — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (manusa) [reply CHANGELOG.md:53]: Yes, probably

**Comment by manusa** [comment]
The functionality introduced in this PR can be seen in action in the following YouTube video:

https://youtu.be/_sf5RN1afNM?t=26

**Pull request merged**